interfaces/builtin/optical_drive.go

```diff
@@ -30,8 +30,8 @@ const opticalDriveBaseDeclarationSlots = `
 
 const opticalDriveConnectedPlugAppArmor = `
 # Allow read access to optical drives
-/dev/sr[0-9]* r,
-/dev/scd[0-9]* r,
+/dev/sr[0-9]* rw,
+/dev/scd[0-9]* rw,
```

Comment:
These policy changes look fine.

Reply:
Change made as requested.

I like the idea of keeping auto connection limited to read-only use, and only allowing writable connections via the discretion of the device owner. Adding a `writable` attribute sounds like a sane way to achieve this. I'm not familiar with the inner-workings of snapd enough (read: at all) to say how easy or difficult, or the appropriateness of, that would be.